fetch.js

```diff
@@ -267,13 +267,16 @@
 
   function headers(xhr) {
     var head = new Headers()
-    var pairs = xhr.getAllResponseHeaders().trim().split('\n')
```

Comment:
Sure. I was just trying to avoid creating and `forEach`-ing that array for nothing.